### DISCUSSION TITLE
Should InMemoryBuffer be renamed?

### DISCUSSION THREAD

**Discussion by MartinZikmund** [unoplatform/uno#4966] [Q&A]
## What would you like to be added:

I have noticed Uno has an `InMemoryBuffer` public class, which is used in several places in the repo. UWP does not have this class, but has a `Buffer` class, which seems to fulfill the same purpose. Should `InMemoryBuffer` be renamed as part of breaking change roundup to `Buffer`? It is unlikely anyone depends on it directly, as it can be constructed only internally and to the outside world always surfaced as `IBuffer`.

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by jeromelaban** [comment]
It does not need to be public, indeed. There's no point at exposing it at this point.

**Comment by MartinZikmund** [comment]
I will probably make a copy of the class to implement `Buffer` class which is part of the UWP, and then the `InMemoryBuffer` class can be removed as part of breaking changes.
